src/NuGet.Clients/NuGetConsole.Host.PowerShell/RegistryHelper.cs

```diff
@@ -24,11 +25,9 @@ public static bool CheckIfPowerShell2OrAboveInstalled()
                 return true;
             }
 
-            // PS 3.0 and 4.0 is set under "...\PowerShell\3" key
+            // PS 3.0 and higher is set under "...\PowerShell\3" key
             keyValue = GetSubKeyValue(@"SOFTWARE\Microsoft\PowerShell\3\PowerShellEngine", "PowerShellVersion");
-            if ("3.0".Equals(keyValue, StringComparison.OrdinalIgnoreCase)
-                ||
-                "4.0".Equals(keyValue, StringComparison.OrdinalIgnoreCase))
+            if (Version.TryParse(keyValue, out Version result) && result >= new Version("3.0"))
```

Comment:
nit: you could save the extra string parsing of ``"3.0""`` with ``&& result.Major >= 3`` and it would be the same check